docs/pages/enroll-resources/desktop-access/active-directory.mdx

```diff
@@ -620,6 +628,47 @@ To configure Teleport to protect access to Windows desktops:
    For a detailed description of the configuration fields, see
    [Desktop Configuration Reference](../../reference/agent-services/desktop-access-reference/configuration.mdx).
 
+1. Configure LDAP server selection.
+
+   To find desktops to connect to in Active Directory, Teleport must be given the
+   address of an LDAP server. There are two ways to specify the LDAP server, either
+   by providing a single LDAP server address in the `addr` field, or by enabling
+   `locate_server` to automatically discover the LDAP server from DNS SRV records.
+
+   If you opt to use `addr`, for the best results, the address should point to a
+   highly-available endpoint to avoid downtime. The port for LDAPS should also be
+   included, which is typically 636.
+
+   <details>
+   <summary>Example of using the addr field</summary>
+   ```yaml
+   windows_desktop_service:
+      enabled: true
+      ldap:
+         addr: ldap.example.com:636
+         domain: example.com
+   ```
+   </details>
+
+   Alternatively, you can enable `locate_server` to automatically discover an
+   LDAP server from the DNS SRV records of your specified `domain`. This is useful
+   if you have multiple LDAP servers for high availability and load balancing.
+   Optionally, you can specify a `site` to limit the discovery to a specific
+   Active Directory site.
+
+   <details>
+   <summary>Example of using locate_server</summary>
+   ```yaml
+   windows_desktop_service:
+      enabled: true
+      ldap:
+         locate_server: 
+            enable: true
```

Comment:
Is it `enable` or `enabled`? This PR switches back and forth between the two.

Reply:
Should be enabled, I've fixed it now